README.md

```diff
@@ -26,7 +26,6 @@
 
 [delta]
     navigate = true  # use n and N to move between diff sections
-    dark = true      # or light = true, or omit for auto-detection
```

Comment:
Dan actually re-added this line on purpose, for some terminals the auto-detection takes too long.

Reply:
Sorry, I think this was a bad rebase. Let me force push with the actual changes.